src/pages/artikler/[...slug].astro

```diff
@@ -27,6 +27,12 @@ const authorNames = authors
   .join(', ');
 const humanDate = formatHumanDate(entry.data.date);
 
+const imageData: ImageData = {
+  width: entry.data.image.width, 
+  height: entry.data.image.height
+}
+const scaledImage = scaleImage(imageData, 1070, 4/3);
+
```

Comment:
I think maybe we can send the entire image object instead of making a new type called imageData, no?